### PR TITLE
ampt comparison

### PR DESCRIPTION
**Pull request by davemooreuws**

### CODE REVIEW

File: src/pages/faq/comparison/ampt.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+These are the main differences between the two:
+
+1. **Cloud Provider Support:** Nitric is provider-agnostic, capable of deploying to multiple cloud providers such as AWS, Google Cloud, and Azure. Ampt is primarily designed for AWS services.
+2. **Language Support:** Nitric provides multiple libraries for TypeScript/JavaScript, Python, Go, C# .NET and Java, allowing developers to choose their preferred language. Currently Ampt only supports TypeScript and JavaScript.
+3. **Infrastructure Provisioning:** Nitric uses Pulumi by default for provisioning cloud resources, but also allows the use of [custom providers](/reference/providers/custom/building-custom-provider). Ampt uses a hosted provisioning engine via their online platform.
+4. **Local Simulation:** Nitric provides a fully offline local cloud simulation tool, allowing you to test your applications locally before deploying them. This is a feature that Ampt does not offer, for local development Ampt deploys and updates your application to the cloud via a sandbox environment.
+5. **Open Source:** As an open-source framework, Nitric does not require a platform or subscription to deploy applications.
```
Comment (HomelessDinosaur) [inline src/pages/faq/comparison/ampt.mdx:12]: Would it be beneficial to add a comment about Ampt being closed source? Currently its implied, not stated explicitly.

**Review by HomelessDinosaur — CHANGES_REQUESTED** [review]

File: src/pages/faq/comparison/ampt.mdx (hunk 1 above)
Comment (davemooreuws) [reply src/pages/faq/comparison/ampt.mdx]: Changed

**Review by HomelessDinosaur — APPROVED** [review]

File: src/pages/faq/comparison/ampt.mdx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+export const description = 'Comparison to Ampt'
+
+# Ampt vs. Nitric
+
+Nitric is a framework that empowers developers to build complete cloud applications on their preferred cloud platform, combining infrastructure declaration and application code in one place. On the other hand, Ampt is a development platform tailored for building cloud backend applications using the JavaScript/TypeScript language, with deployment options only for AWS. Both serve the purpose of simplifying cloud application development but operate in slightly different ways and cater to different specific needs.
```
Comment (jyecusch) [inline src/pages/faq/comparison/ampt.mdx:5]:
```suggestion
Nitric is a framework that empowers developers to build complete cloud applications on their preferred cloud platform, combining infrastructure declaration and application code in one place. Ampt is a development platform tailored for rapidly building cloud applications using JavaScript or TypeScript. Both serve the purpose of simplifying cloud application development but operate in different ways and cater to different specific needs.
```

File: src/pages/faq/comparison/ampt.mdx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+These are the main differences between the two:
+
+1. **Cloud Provider Support:** Nitric is provider-agnostic, capable of deploying to multiple cloud providers such as AWS, Google Cloud, and Azure. Ampt is primarily designed for AWS services.
```
Comment (jyecusch) [inline src/pages/faq/comparison/ampt.mdx:8]:
```suggestion
1. **Cloud Provider Support:** Nitric is provider-agnostic, capable of deploying to multiple cloud providers such as AWS, Google Cloud, and Azure. Ampt is a platform, which currently runs on AWS.
```

File: src/pages/faq/comparison/ampt.mdx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+2. **Language Support:** Nitric provides multiple libraries for TypeScript/JavaScript, Python, Go, C# .NET and Java, allowing developers to choose their preferred language. Currently Ampt only supports TypeScript and JavaScript.
```
Comment (jyecusch) [inline src/pages/faq/comparison/ampt.mdx:9]:
```suggestion
2. **Language Support:** Nitric provides multiple libraries for TypeScript/JavaScript, Python, Go, C# .NET and Java, allowing developers to choose their preferred language. Currently Ampt currently supports TypeScript and JavaScript.
```

File: src/pages/faq/comparison/ampt.mdx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+3. **Infrastructure Provisioning:** Nitric uses Pulumi by default for provisioning cloud resources, but also allows the use of [custom providers](/reference/providers/custom/building-custom-provider). Ampt uses a hosted provisioning engine via their online platform.
```
Comment (jyecusch) [inline src/pages/faq/comparison/ampt.mdx:10]:
```suggestion
3. **Infrastructure Provisioning:** Nitric uses Pulumi by default for provisioning cloud resources, but also allows the use of [custom providers](/reference/providers/custom/building-custom-provider). Ampt uses a hosted provisioning engine via their online platform. While resources deployed with Nitric are statically determined at deployment, Ampt is able to reallocate resources between AWS services at runtime to optimize applications.
```

File: src/pages/faq/comparison/ampt.mdx [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+4. **Local Simulation:** Nitric provides a fully offline local cloud simulation tool, allowing you to test your applications locally before deploying them. This is a feature that Ampt does not offer, for local development Ampt deploys and updates your application to the cloud via a sandbox environment.
```
Comment (jyecusch) [inline src/pages/faq/comparison/ampt.mdx:11]:
```suggestion
4. **Local Simulation:** Nitric provides a fully offline local cloud simulation tool, allowing you to test your applications locally before deploying them. This is a feature that Ampt does not offer. However, for local development Ampt deploys and syncs your application to the cloud via a sandbox environment, allowing for rapid testing of local changes.
```

File: src/pages/faq/comparison/ampt.mdx [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+5. **Open Source:** As an open-source framework, Nitric does not require a platform or subscription to deploy applications, while closed-source Ampt requires sign-up and may involve additional costs.
```
Comment (jyecusch) [inline src/pages/faq/comparison/ampt.mdx:12]:
```suggestion
5. **Open Source:** As an open-source framework, Nitric does not require a platform or subscription to deploy applications. Ampt is a hosted platform which requires an account and provides both free and paid offerings.
```

File: src/pages/faq/comparison/ampt.mdx [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+</CodeGroup>
+
+## Differences
+
+|                      | Nitric                                                                                       | Ampt                                     |
+| -------------------- | -------------------------------------------------------------------------------------------- | ---------------------------------------- |
+| Language             | Your choice                                                                                  | JavaScript / TypeScript                  |
+| Lines of code        | 9                                                                                            | 9                                        |
+| Cloud Infrastructure | Inferred                                                                                     | Inferred                                 |
```
Comment (jyecusch) [inline src/pages/faq/comparison/ampt.mdx:52]:
```suggestion
| Cloud Infrastructure | Inferred, Static                                                                                     | Inferred, Dynamic                                 |
```

File: src/pages/faq/comparison/ampt.mdx [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+| Extensibility        | [Custom providers](/reference/providers/custom/building-custom-provider) can be created      | None                                     |
+| Local simulation     | Built-in local simulator with instant hot reloading and offline support                      | No, provides proxy to a deployed sandbox |
```
Comment (jyecusch) [inline src/pages/faq/comparison/ampt.mdx:54]:
```suggestion
| Local development     | Built-in local simulator with instant hot reloading and offline support                      | Active project synchronization to a cloud sandbox with a local proxy |
```

File: src/pages/faq/comparison/ampt.mdx [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+| Cloud providers      | AWS, Azure, GCP and [Custom providers](/reference/providers/custom/building-custom-provider) | AWS                                      |
```
Comment (jyecusch) [inline src/pages/faq/comparison/ampt.mdx:55]:
```suggestion
| Cloud providers      | AWS, Azure, GCP and [Custom providers](/reference/providers/custom/building-custom-provider) | Ampt Platform (AWS)                                      |
```

**Review by jyecusch — CHANGES_REQUESTED** [review]

File: src/pages/faq/comparison/ampt.mdx (hunk 4 above)
Comment (davemooreuws) [reply src/pages/faq/comparison/ampt.mdx:9]: I think this is redundant, just need one currently

**Review by jyecusch — APPROVED** [review]

**Pull request merged**